s3po/backends/s3.py

```diff
@@ -93,3 +93,9 @@ def func():
             multi.upload_part_from_file(StringIO(data), count)
             multi.complete_upload()
             return True
+
+    def list(self, bucket, prefix=None, delimiter=None, headers=None):
```

Comment:
Ah, ok, I forgot about that. Boo, `swift`! Yeah, I'm on board with a `limit` of some sort. Maybe as a required parameter? Or perhaps better yet, we could make it iterable ourselves and just chain the calls?